app/models/katello/content_view_version.rb

```diff
@@ -73,8 +73,8 @@ def self.with_library_repo(repo)
 
     def self.for_version(version)
       major, minor = version.to_s.split('.')
-      query = where(:major => major)
-      query.where(:minor => minor) if minor
```

Comment:
I think the reason this was broken up into two queries is so one can do Katello::ContentViewVersion.for_version("1") and get all the content view versions with a major column of 1, so 1.0, 1.1, 1.2, etc... maybe @jlsherrill or @ehelms could confirm this.

When assign minor to be zero, it won't function this way. if you add a "query =" here, this should fix the bug, as the minor version query isn't doing anything here because it isn't being assigned back to 'query'.

in example
```       
    def self.for_version(version)
        major, minor = version.to_s.split('.')
        query = where(:major => major)
        query = query.where(:minor => minor) if minor
        query
      end
```